src/less/section-notice/base/section-notice.less

```diff
@@ -84,3 +84,9 @@ span[role="region"].section-notice {
         width: auto;
     }
 }
+
+[dir="rtl"] {
+    .icon--confirmation-filled {
```

Comment:
Add `.section-notice__header` scope here. Also should apply to all 3 types of icon